javascript/sdk/yarn.lock

```diff
@@ -0,0 +1,242 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
remove + add to .gitignore

Reply:
Would love to know your reasoning as to why later on